README.md

```diff
@@ -216,12 +216,21 @@ endpointUrl:
 ```
 
 ## Deploying
-To deploy to a docker registry there are two options.
-1. Edit `deploy.bash` to set the docker registry and docker wrapper for the external server.
-2. Export the 2 configurations by running:
+There are two ways to configure the build system to deploy the CDR service to a cloud:
+- Edit the`deploy.bash` file. Set two variables near the top of the file:
+  - Set `SET_DOCKER_REG="ip:port"`
+  - Set `SET_DOCKER_W=/path/to/docker/wrapper/"`
```

Comment:
✏️ Remove `Set ` for these 2 lines because the instruction above already includes that instruction